boards/arm/nucleo_wb55rg/nucleo_wb55rg.dts

```diff
@@ -77,6 +77,10 @@ arduino_i2c: &i2c1 {
 	status = "okay";
 };
 
+&spi1 {
```

Comment:
Can you add it as arduino_spi? (cf arduino_serial)

Reply:
changed